src/main.js

```diff
@@ -41,9 +41,17 @@ var currentCover;
 
 // Add your event listeners here 👇
 
+window.onload = newRandomCover();
```

Comment:
Thank you for finding the "onload" method. Maybe once we start adding more event listeners, we can change the functionality to match up with the button click interaction?